bundles/net.tourbook/src/net/tourbook/messages.properties

```diff
@@ -3462,6 +3465,8 @@ pref_appearance_number_of_recent_tags         = Number of recent &tags:
 pref_appearance_number_of_recent_tags_tooltip = Number of recently used tags which are displayed in the context menu.\n\
                                                 \n\
                                                 0 will hide recently used tags.
+pref_appearance_pace_computation_option         = Pace computation options:
```

Comment:
A label should not have a colon

Reply:
I will fix that.

Reply:
Fixed. And I also fixed the label `pref_appearance_number_of_recent_tags` above as it has also a colon